osmosis-1/osmosis.zone_assets.json

```diff
@@ -1517,6 +1517,13 @@
       "path": "transfer/channel-874/untrn",
       "osmosis_main": false,
       "osmosis_frontier": true
+    },
+    {
+      "chain_name": "quasar",
+      "base_denom": "uqsr",
+      "path": "transfer/channel-688/uqsr",
+      "osmosis_main": true,
```

Comment:
I haven't been able to confirm this. For the time being can we set osmosis_main to false, please?